src/components/screens/new-questionnaire-screen.tsx

```diff
@@ -181,62 +218,65 @@ const NewQuestionnaireScreen = () => {
-            mt: 4,
-            width: "100%",
-            display: "flex",
-            flexDirection: "column",
-            height: "100"
-          }}
-          >
-            <CardActions
             sx={{
+              p: 2,
+              mt: 4,
+              width: "100%",
               display: "flex",
-              justifyContent: "flex-end",
-              padding: 0,
-              alignItems: "flex-start",
-              flexDirection: { xs: "column", sm: "row" },
-              width: "100%"
+              flexDirection: "column"
             }}
           >
-            <Box sx={{ display: "flex", flexDirection: "column", width: "70%", mr: 4 }}>
-              <Typography
-                variant="h6"
-                gutterBottom
-                sx={{ display: "flex", alignItems: "center", mb: 1, mt: 1 }}
-              >
-                {strings.newQuestionnaireScreen.countedAnswers} {countCorrectAnswers()}
-              </Typography>
-              <Typography variant="h6" gutterBottom sx={{ mb: 1, mt: 1 }}>
-                {strings.newQuestionnaireScreen.requiredAnswers} {questionnaire.passScore}
-              </Typography>
-              <Slider
-                value={questionnaire.passScore}
-                onChange={handlePassScoreSliderChange}
-                step={1}
-                marks
-                min={0}
-                max={countCorrectAnswers()}
-                valueLabelDisplay="auto"
-                sx={{ mt: 1, mb:1, width: "70%" }}
-              />
-            </Box>
-            <Button
-              sx={{ display: "flex", alignItems: "center", mt: 6, mr: 4 }}
-              id="save-submit"
-              size="large"
-              variant="contained"
-              color="success"
-              onClick={saveQuestionnaire}
-              disabled={loading || checkBeforeSave}
+            <CardActions
+              sx={{
+                display: "flex",
+                justifyContent: "flex-end",
+                padding: 0,
+                alignItems: "flex-start",
+                flexDirection: { xs: "column", sm: "row" },
+                width: "100%"
+              }}
             >
-              {loading ? (
-                <CircularProgress size={24} color="inherit" />
-              ) : (
-                strings.newQuestionnaireScreen.saveButton
-              )}
-            </Button>
+              <Box sx={{ display: "flex", flexDirection: "column", width: "70%", mr: 4 }}>
+                <Typography
+                  variant="h6"
+                  gutterBottom
+                  sx={{ display: "flex", alignItems: "center", mb: 1, mt: 1 }}
+                >
+                  {strings.newQuestionnaireScreen.countedAnswers} {countCorrectAnswers()}
+                </Typography>
+                <Typography variant="h6" gutterBottom sx={{ mb: 1, mt: 1 }}>
+                  {strings.newQuestionnaireScreen.requiredAnswers} {questionnaire.passScore}
```

Comment:
This could be done using the formatting strings option which allows us to pass the variable to the string. It's been done in other places in this project e.g. for the questionnaireTable.confirmDeleteTitle string.
Not essential in this case however